validator/write_api.py

```diff
@@ -201,3 +188,28 @@ def set_default_feature_weights_id():
             "feature_weight_set_id": default_id,
         }
     )
+
+
+@bp.route("/datasets/books/<vuid>/feature_weights_id", methods=["PUT"])
```

Comment:
Hmm, this reminds me: read_api needs a matching "GET" method, to return a given books fw_id

Reply:
Good job on the GET method. :)